README.md

```diff
@@ -49,7 +66,7 @@ Environment: NVIDIA RTX A6000 with Intel Xeon Silver 4314 on Ubuntu 22.04.3
 
 Baseline: We utilized the widely adopted [Alpaca-LoRA](https://github.com/tloen/alpaca-lora) as a foundation. On a single GPU, we independently ran multiple Alpaca-LoRA processes in parallel (marked as *Baseline@SYNC*) and sequentially (marked as *Baseline@SEQ*), forming two baseline methods for the experiments.
 
-#### Training Latency
+#### Training Latency and Throughput
```

Comment:
do we need to mention that we test this on A100, and rest of results are based on the same GPU configure?